types/carbon-components-react/index.d.ts

```diff
@@ -1,7 +1,7 @@
 // Type definitions for carbon-components-react 7.7
 // Project: https://github.com/carbon-design-system/carbon/tree/master/packages/react
 // Definitions by: Kyle Albert <https://github.com/kalbert312>
-//                 Sebastien Gregoire <https://github.com/sgregoire>
```

Comment:
Please, do not remove other contributors

Reply:
Hi @sgregoire , sry 🙏was without intention. Fixed it though.